### ISSUE TITLE
Priduct/Package

### ISSUE THREAD

**Issue by astvoip** [iNextrix/ASTPP#526]
Hi.
I installed ASTPP4.0.
In the products I have created a 300 min free package with several destinations.
The 300 min free package does not work. The system works with the correct payment for destinations that have 300 min free.
Can you please help me?

**Comment by astvoip** [comment]
The system works all the time with the payment, It does not bring the package with free minutes installed in the client account

**Comment by Hemdip** [comment]
Please confirm that you have selected prefix/destination code in package and share result. Because it seems just configuration issue

**Comment by astvoip** [comment]
![Screenshot from 2019-06-24 11-33-04](https://user-images.githubusercontent.com/52051496/60008258-112c4f00-9674-11e9-96a8-78a1924e336f.png)
![Screenshot from 2019-06-24 11-33-14](https://user-images.githubusercontent.com/52051496/60008277-1be6e400-9674-11e9-94ca-ee85f7b8a958.png)
Here you have the 300 min free configuration for several destinations.
The package does not work.

**Comment by astvoip** [comment]
![Screenshot from 2019-06-24 12-27-54](https://user-images.githubusercontent.com/52051496/60012135-851e2580-967b-11e9-9de4-b452b8b76308.png)

Here is the call with this destination, always with payment.

**Comment by astvoip** [comment]
![Screenshot from 2019-06-24 20-07-41](https://user-images.githubusercontent.com/52051496/60041473-ce8e6500-96bb-11e9-806a-b22f3113358c.png)
Fro un ather configuration. Here you have the 300 min free configuration for several destinations.
The package does not work.

**Comment by astvoip** [comment]
I've tried all the possibilities.
Result? Same

**Comment by Hemdip** [comment]
@astvoip As per your screenshot it seems like your have set different prefix (code) in package and you dial different prefix number and due to that it going as Standard call.

**Comment by astvoip** [comment]
This is not true. Destinations are with different prefixes. Have you tried this package? Can you share your configuration? I've tried all the possibilities she does not work, I'm sure of that.

**Comment by astvoip** [comment]
Any response???

**Comment by hardikinextrix** [comment]
@astvoip 

- Package should create with correct prefixes and information.
- Package should be added to customer profile.
- Prefix should be match with origination rates and prefixes.
In your case we found that the number you are dialing is not added under the package destinations. that can be the reason package is not applying. 
Also confirm package should be added correctly on customer profile. reason is you have selected rate group at the time of package creation but the option "apply on existing account" is set as "NO" so please confirm.

**Comment by astvoip** [comment]
Thank you.
In the manner that you describe the packages with several destinations function correctly.
When the number is called 35569 and does not translate the package with that destination it works.
The problem is when the number is translated, for example:
When the destination is called as 0035569 and the system translates as 35569, the package with these destinations does not work.
It passes on the pay packet.
So when the call is translated it does not work on the package created.
So to have this package we do not have to translate the calls, that makes no sense because the translation of the numbers is always in function.
Do you have a solution?

**Comment by hardikinextrix** [comment]
Generally customer side translation are affect the rates so it will be affected to package also but the gateway side translation will not affect it.

**Comment by astvoip** [comment]
Mr. Hardik.
Thank you for your suggestion. We know we are talking in skype.
The problem is when the package affects the case the customer's call is translated. The customer's call must be translated because he can call in some form, for example: 069, 0035569,69, + 35569 all of these translates to standard 35569 always. We do not educate the client to make calls only in a way. In this case, the package has failed in version 4.0. It does not have value is not functional. In version 3.6 it does not constitute a problem. You have to fix it so that it does not affect when we translate client calls.
Thank you for your correction
